docs/getting-started/index.rst

```diff
@@ -17,19 +17,30 @@ Getting started
 Installation
 ------------
 
+There are various ways for users to install Gammapy. We recommend setting up an
+environment using either conda or mamba. Here are two methods to quickly install Gammapy.
+
 .. panels::
-    :column: col-lg-6 col-md-6 col-sm-12 col-xs-12 p-3
+    :column: col-12 p-3
 
     Working with conda?
     ^^^^^^^^^^^^^^^^^^^
 
-    Gammapy can be installed with `Anaconda <https://docs.continuum.io/anaconda/>`__ or Miniconda:
+    First set up and activate your environment:
```

Comment:
I would not show this here. Creating virtual environments and installing the software are two different things. I would keep the current panels and just add a note on the recommendation of the using a virtual environment like:

![Screenshot 2024-02-28 at 10 40 50 AM](https://github.com/gammapy/gammapy/assets/3715928/82504814-d93d-43c7-aa12-d6d79cf00af5)